`intel/test_block_load.py::test_block_load_dpas_layout[True-int8-128-8]` fails on B580, Linux

```
2025-04-15T15:33:12.8935313Z FAILED intel/test_block_load.py::test_block_load_dpas_layout[True-int8-128-8] - AssertionError: assert (True and False)
2025-04-15T15:33:12.8936098Z  +  where True = <built-in method equal of type object at 0x724b56d59c00>(tensor([[  35,  -63,   57,  ...,  -95, -102,  -78],\n        [-101, -124, -115,  ...,   -3,  -78,  -25],\n        [  30,   52,  127,  ...,   94,   29,  -90],\n        ...,\n        [-119,  -33,   40,  ...,    8,  -19,   40],\n        [  19,  -29,  113,  ...,   74,   52,    7],\n        [  45,   -8,  109,  ...,   65,  125,   93]], device='xpu:0',\n       dtype=torch.int8), tensor([[  35,  -63,   57,  ...,  -95, -102,  -78],\n        [-101, -124, -115,  ...,   -3,  -78,  -25],\n        [  30,   52,  127,  ...,   94,   29,  -90],\n        ...,\n        [-119,  -33,   40,  ...,    8,  -19,   40],\n        [  19,  -29,  113,  ...,   74,   52,    7],\n        [  45,   -8,  109,  ...,   65,  125,   93]], device='xpu:0',\n       dtype=torch.int8))
2025-04-15T15:33:12.8936783Z  +    where <built-in method equal of type object at 0x724b56d59c00> = torch.equal
2025-04-15T15:33:12.8937511Z  +  and   False = <built-in method equal of type object at 0x724b56d59c00>(tensor([[  21,  -60,  -27,  ...,  -91,  -97,  103],\n        [-121, -107,   24,  ...,   83, -124,   62],\n        [ 113,   68,  -34,  ...,  -52, -109,   11],\n        ...,\n        [  93,    7,  -32,  ..., -122,   85,  117],\n        [ -63,  -81,   18,  ..., -101,   17,  -35],\n        [  62,  105,   82,  ...,   70,    7,   63]], device='xpu:0',\n       dtype=torch.int8), tensor([[  21,  -27,   19,  ...,   77,    2,   31],\n        [-121,   24,   -5,  ...,   78,    8,   30],\n        [ 113,  -34,  124,  ...,   85,   91,  116],\n        ...,\n        [  78,   69, -101,  ...,  -81,    9,  -48],\n        [ 108,  -95,   71,  ..., -116,   67,   91],\n        [ -98, -111,   71,  ...,  -64,   56,   78]], device='xpu:0',\n       dtype=torch.int8))
2025-04-15T15:33:12.8938183Z  +    where <built-in method equal of type object at 0x724b56d59c00> = torch.equal
```

```
2025-04-15T15:33:12.8932819Z ----------------------------- Captured stderr call -----------------------------
2025-04-15T15:33:12.8933317Z /tmp/pytest-of-runner/pytest-5/popen-gw3/test_block_load_dpas_layout_Tr2/test_block_load_dpas_layout.ttgir:12:18: error: 'triton_gen.2Dblockload' op result size of 256 bits does not match the expected size of 128 bits
2025-04-15T15:33:12.8933932Z             %2 = tt.load %1 {boundaryCheck = array<i32: 0, 1>, triton_intel_gpu.block_io = "row_major"} : !tt.ptr<tensor<128x8xi8, #ttg.dot_op<{opIdx = 0, parent = #mma, kWidth = 2}>>>
2025-04-15T15:33:12.8934247Z                  ^
```

https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/14472043222

--- a/python/test/unit/intel/test_block_load.py
+++ b/python/test/unit/intel/test_block_load.py
@@ -16,6 +16,9 @@ from triton._internal_testing import is_xpu
 @pytest.mark.xfail(not torch.xpu.get_device_capability()['has_subgroup_2d_block_io'],
                    reason="Block loads not supported on this architecture")
 def test_block_load_dpas_layout(M, N, dtype_str, transpose, device, tmp_path: pathlib.Path):
+    if transpose and N == 8:
+        pytest.xfail("Pitch = 8 is not allowed by block IO")
+
     # modify the layouts to ensure the correct OCL/SPIRV intrinsic is called for each datatype
     if dtype_str == "int8":
         A_width = 2

--- a/scripts/skiplist/xe2/intel.txt
+++ b/scripts/skiplist/xe2/intel.txt
@@ -1,2 +1,0 @@
-# https://github.com/intel/intel-xpu-backend-for-triton/issues/3935
-python/test/unit/intel/test_block_load.py::test_block_load_dpas_layout[True-int8-128-8]

